analyzer/src/main/kotlin/PackageManager.kt

```diff
@@ -318,6 +319,11 @@ abstract class PackageManager(
         }
     }
 
+    /**
+     * Get the [Options] from the [PackageManagerConfiguration] for this [package manager][managerName].
+     */
+    protected fun getOptions(): Options = analyzerConfig.getPackageManagerConfiguration(managerName)?.options.orEmpty()
```

Comment:
How about making this take an optional key-argument, so one could write `getOptions(OPTION_DIRECT_DEPENDENCIES_ONLY)` instead of `getOptions()[OPTION_DIRECT_DEPENDENCIES_ONLY]`? Or make it a property?

Reply:
I have made it a property.